src/libsync/pushnotifications.h

```diff
@@ -0,0 +1,68 @@
+#ifndef PUSHNOTIFICATIONS_H
+#define PUSHNOTIFICATIONS_H
+
+#include <QWebSocket>
+
+#include "capabilities.h"
+
+class TestPushNotifications;
+
+namespace OCC {
+
+class Account;
+class AbstractCredentials;
+
+class PushNotifications : public QObject
+{
+    Q_OBJECT
+
+public:
+    PushNotifications(Account *account);
+
+    /**
+     * Setup push notifications
+     *
+     * This method needs to be called before push notifications can be used.
+     */
+    void setup();
+
+    /**
+     * Reset push notifications
+     *
+     * After a call to this function the object will be in a state like after a call to setup.
+     */
+    void reset();
+
+    /**
+     * Get the types of push notifications available
+     */
+    PushNotificationTypes pushNotificationsAvailable() const;
```

Comment:
This is "just" forwarding from Capabilities which is available in FolderMan. What is calling into this in practice?